core/src/App.html

```diff
@@ -258,18 +256,6 @@
           builderCompatibilityMode: Boolean(window.builderCompatibilityMode),
           isolateAllViews
         };
-        window.addEventListener('popstate', async e => {
```

Comment:
The 404 error alert was supposed to be visible until the route change. Currently, it's visible until you close it manually.

Reply:
This is on purpose. SAP Fiori Fundamental is not providing automatic dismiss in their guidelines, so the user will always have to close it by himself